backend/remote/backend_common.go

```diff
@@ -299,6 +295,7 @@ func (b *Remote) costEstimate(stopCtx, cancelCtx context.Context, op *backend.Op
 			deltaRepr := strings.Replace(ce.DeltaMonthlyCost, "-", "", 1)
 
 			if b.CLI != nil {
+				b.CLI.Output(b.Colorize().Color(msgPrefix + ":\n"))
```

Comment:
I moved the specific `msgPrefix` part here and the other lines below because other cases have a different prefex mesage, and there wasn't a clean way to not duplicate.